cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -225,6 +225,7 @@ func TestPolicyUpdate(t *testing.T) {
 		krb5ccname            string
 		krb5ccNamesState      []krb5ccNamesWithState
 		clearDirs             []string // Removes already generated system files eg dconf db, apparmor profiles, ...
+		addPaths              []string
```

Comment:
shouldn’t that be `addDir` (as we always append / to the path name)

Reply:
I think we can do both ways. I can either change the name to be addDirs or I could make it a []struct{path, isDir} and we append / to the path if isDir is true. Wdyt?
For some reason, filepath.Join trims the last / that it's on the string, so if we want to be able to handle directories and files, we must do something like the []struct[}

Reply:
I think addDir is the more consice and declares precisely what we do here.

Reply:
It is, indeed. But is less versatile as well...

Reply:
Right, but the question is: do we need versatility right now? Do we have other use cases matching another need where this field should be versatile? Is it part of any public API making this design decision set in stone forever?

If thhe answer to the 3 questions is "no" (spoiler alert: it is!), then let’s not plan for a future that may (and probably will) never happen. :)